### DISCUSSION TITLE
Port forwarding using vs code's remote tunnel results in blank page

### DISCUSSION THREAD

**Discussion by JensvandeZande** [dagster-io/dagster#19338] [Q&A] [answered]
Dear all,

I am using my desktop as my compute and I use my laptop to remotely develop on my desktop. I am using VsCode's remote tunnel functionality to remotely connect to my desktop. Using port forwarding, I can then open up a service that is running on my desktop by navigating to `localhost:some_port` in my browser on my laptop. This all works fine when I spin up a very simple flask app. However, when trying with dagster I get a blank page. Can you guys help me out in making this work?

To test it I just used the `dagster dev` command which opens a web server on port 3000.

KR Jens

**Comment by alangenfeld** [comment]
Are you forwarding port 3000 ? 
https://code.visualstudio.com/docs/editor/port-forwarding

**Reply by JensvandeZande** [reply]
@alangenfeld yes I am.

**Reply by alangenfeld** [reply]
If you open the browser dev tools on the blank page does it show any errors or specific resources its failing to fetch?

**Reply by JensvandeZande** [reply]
Hi @alangenfeld,

This is what I see when I inspect the page. Should I look for something more specific?

![image](https://github.com/dagster-io/dagster/assets/49940005/8ce0e396-eeca-4c2e-b625-6c25a40f13fe)

**Reply by JensvandeZande** [reply]
Hi @alangenfeld, 

This is what I see when opening the console in inspect mode on my desktop which host the webserver. 

![image](https://github.com/dagster-io/dagster/assets/49940005/269fef19-4e1c-44c8-9be8-dfabea12393a)

**Reply by alangenfeld** [reply]
For the case when its not working, the `console` tab might have JS errors in it and the `network` tab may reveal which specific resources that are failing to fetch. Based on the fact that you are getting a blank page means that `index.html` is fetching successfully (otherwise you would get a `This site can’t be reached` browser error).

**Reply by JensvandeZande** [reply]
Hi @alangenfeld,

On my laptop, it shows no console and network logs or metrics. See pictures below
![image](https://github.com/dagster-io/dagster/assets/49940005/825f48ad-bb6b-4b87-81fe-aefc355df217)
![image](https://github.com/dagster-io/dagster/assets/49940005/3b9a38e4-9e82-4586-a977-37e830900943)



In the last picture of my previous comment you can see the console logs of my desktop.

KR Jens

**Reply by alangenfeld** [reply]
You have to reload the page with the network tab open to get the recorded activity to show up. That said its the information of what resources are failing to load may not generate any useful leads.

**Reply by JensvandeZande** [reply]
Hi @alangenfeld,

When reloading the page this is what I get from the network tab. Console remains blank, even after reloading the page.

![image](https://github.com/dagster-io/dagster/assets/49940005/8934ce42-e596-469f-bc13-b1553d894b3f)

**Reply by alangenfeld** [reply]
Bizarre, I don't have a good explanation for how you are getting a blank page despite all the resources loading successfully. I'm stumped on this one.

**Reply by JensvandeZande** [reply]
Hi @alangenfeld,

That is unfortunate. Thanks for your efforts! Could you please ask one of your fellow colleagues/maintainers?

KR Jens

**Reply by garethbrickman** [reply, marked as answer]
May be worth opening localhost inside an Incognito browser tab just to eliminate any extensions or other non-default settings in your regular browser profile as possible causes

**Reply by JensvandeZande** [reply]
Omg yes! That was it! I feel kind of stupid I did think of that myself haha. Many thanks @garethbrickman ! I guess I should now one by one remove the extensions to see which one is causing the error, or is there another approach to see which one is the culprit? 

KR Jens
